src/fits.cc

```diff
@@ -720,12 +720,11 @@ void writeKeyFromProperty(
         }
     } else {
         // FIXME: inherited this error handling from fitsIo.cc; need a better option.
-        pex::logging::Log::getDefaultLog().log(
-            pex::logging::Log::WARN,
+        LOGLS_WARN("afw.writeKeyFromProperty",
             makeErrorMessage(
                 fits.fptr, fits.status,
-                boost::format("In %s, unknown type '%s' for key '%s'.")
-                % BOOST_CURRENT_FUNCTION % valueType.name() % key
+                boost::format("Unknown type '%s' for key '%s'.")
+                % valueType.name() % key
```

Comment:
log should include the function name. But I didn't mean to drop `BOOST_CURRENT_FUNCTION` here, so I'm adding it back. ( So no changes here)